tests/cephfs/cephfs_recovery/mds_last_seen.py

```diff
@@ -0,0 +1,470 @@
+        )
+
+        # Verify service is started
+        log.info("Verifying that the MDS service is started")
+        out, rc = node.exec_command(
+            sudo=True, cmd="systemctl is-active %s" % service_name, check_ec=False
+        )
+        if "active" in out and rc == 0:
+            log.info(
+                "MDS service %s started successfully (status: %s)"
+                % (service_name, out.strip())
+            )
+            log.info("==== COMPLETED: Restart MDS service ====")
+            return True
+        else:
+            log.warning(
+                "MDS service %s might not be fully started (status: %s)"
+                % (service_name, out.strip())
+            )
+            return False
+
+    except Exception as e:
+        log.error("Error while starting MDS service: %s" % e)
+        log.error(traceback.format_exc())
+        return False
+
+
+def run(ceph_cluster, **kw):
+    """
+    Test MDS last-seen functionality
+
+    Steps:
+    1. Set the standby-replay mds to 2
+    2. Run last-seen for active mds and check if it shows 0s
+    3. Stop MDS service directly using systemctl on the MDS node
+    4. Wait for about 120sec
+    5. Run last-seen for the failed mds and check if the time has passed more than 120sec
+    6. Run last-seen for standby replay mds
+    7. Run last-seen for invalid mds name
+
+    """
+    log.info("====================================================")
+    log.info("===== STARTING: MDS last-seen functionality test =====")
+    log.info("====================================================")
+
+    # Initialize variables that might be referenced in finally block
+    fuse_mount_dir = None
+    kernel_mount_dir = None
+    mds_services_stopped = []  # List to store stopped MDS services
+
+    try:
+        tc = "CEPH-83614244"
+        log.info("Running CephFS tests%s", tc)
+        fs_util = FsUtils(ceph_cluster)
+        clients = ceph_cluster.get_ceph_objects("client")
+        client1 = clients[0]
+        config = kw.get("config", {})  # Add default value
+        if config is None:
+            config = {}
+        build = config.get("build", config.get("rhbuild"))
+        fs_util.prepare_clients(clients, build)
+
+        # Create test filesystem if it doesn't exist
+        log.info("==== STARTING: Create filesystem if it doesn't exist ====")
+        fs_name = "cephfs"
+        out, rc = FsUtils.get_fs_details(client1, fs_name)
+        if fs_name not in out:
+            log.info("Creating filesystem %s" % fs_name)
+            fs_util.create_fs(client1, fs_name)
+        else:
+            log.info("Filesystem %s already exists" % fs_name)
+        log.info("==== COMPLETED: Create filesystem if it doesn't exist ====")
+
+        # Step 1: Set the standby-replay mds to 2
+        log.info("==== STARTING: Step 1 - Configure standby-replay MDS ====")
+        log.info("Setting standby-replay MDS count to 2")
```

Comment:
we have a function to set and validate : 
    def set_and_validate_mds_standby_replay(self, clients, fs_name, boolean):
 -- Please reuse it here...